static/js/click_handlers.js

```diff
@@ -11,7 +11,9 @@ exports.initialize = function () {
     function is_clickable_message_element(target) {
         return target.is("a") || target.is("img.message_inline_image") || target.is("img.twitter-avatar") ||
             target.is("div.message_length_controller") || target.is("textarea") || target.is("input") ||
-            target.is("i.edit_content_button") || target.is(".spoiler-arrow") ||
+            target.is("i.edit_content_button") ||
+            // For spoilers, allow clicking either the header or elements within it (such as the dropdown arrow)
+            target.is(".spoiler-header") || target.parents(".spoiler-header").length > 0 ||
```

Comment:
Could we just delete the spoiler code here?  I'd expect that the click handler for the widget itself, being more tightly bound, would take precedence over this `.messagebox` click handler (like those for most of our other buttons inside messages).

Reply:
If this line is removed, than the click event is absorbed by the select message function, and does not open the block. Interestingly, this does not happen if the event can encompass click behavior but is not `click` (for instance without this line dragging and double clicking are still functional; only single clicks are absorbed). I believe this is because the event selecting for messageboxes is on `#main_div` whereas the spoiler one is on `body` (to support expansion of the spoiler in the formatting help modal).

I'm not sure that stopping propagation on message selection is definitely what you want here, but assuming it is, not having clickable elements select seems like a reasonable workaround to keep them functional (and this code is only extending that workaround, not adding it—the same issue presumably applies to all the other elements selected for here).